StreamChatClient_v3/WebSocketClient/Events/Event.swift

```diff
@@ -19,6 +19,11 @@ protocol EventWithUserPayload: EventWithPayload {
     var userId: UserId { get }
 }
 
+/// A protocol for user owner `Event`s.
+protocol EventWithCurrentUserPayload: EventWithPayload {
+    var currentUserId: UserId { get }
+}
+
```

Comment:
Sorry, but these protocols deserve much better comments. I have no idea why we have them.